aws/v19.0.0-alpha1/announcement.md

```diff
@@ -0,0 +1,5 @@
+**Workload cluster release v19.0.0-alpha1 for AWS is available**. This alpha release introduces Kubernetes 1.24, replaces the use of AWS CNI with Cilium and upgrades most components. **This release is solely to be used for testing in new playground clusters** Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v19.0.0-alpha1/).
```

Comment:
Why only new clusters? you cannot upgrade from the existing cluster?

Reply:
it is Alpha release so no - just for testing out in new clusters

Reply:
beta would be upgradable but we will go for GA straight away